### ISSUE TITLE
Friendly captcha icon missing text alternative - siteimprove issue

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#1234]
Siteimprove is reporting as an accessibility issue that we are missing a text alternative for the friendly captcha vector image. Comment from @fernandolucchesi from Slack:  _This “widget/icon” is implemented by friendlycaptcha, not by us (we import this box from their code). So the ideal would be to create an issue on their codebase so that they update and fix it on their end. Alternatively we can try to add a fix on our side too._

![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/3bbd54df-5d28-4edc-a09b-53ab51e9fdeb)

**Comment by SvSven** [comment]
Should make an issue/PR in the FriendlyCaptcha repository to fix this, then upgrade our dependency once that is merged in and published.

**Comment by padms** [comment]
https://github.com/FriendlyCaptcha/friendly-challenge/issues/133 created an issue here.

**Comment by padms** [comment]
Fix available in [release 0.9.6 ](https://docs.friendlycaptcha.com/#/changelog)
